x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/datafeed/DatafeedJob.java

```diff
@@ -41,6 +43,7 @@
 
     private static final Logger LOGGER = LogManager.getLogger(DatafeedJob.class);
     private static final int NEXT_TASK_DELAY_MS = 100;
+    private static final int BUCKETS_BETWEEN_MISSING_DATA_CHECK = 1;
```

Comment:
Too small? It could be a function of time rather than bucket count i.e check every 10 mins

Reply:
@davidkyle yeah, @dimitris-athanasiou originally brought up the idea of every bucket. 

How long between checks is still up for debate as the true impact on performance is not known.